namui/src/namui/fs/bundle/create_bundle_url.rs

```diff
@@ -0,0 +1,86 @@
+use crate::fs::types::PathLike;
```

Comment:
Moved from fs/util/create_url.rs to fs/bundle/create_bundle_url.
And renamed to create_bundle_url